### PR TITLE
Add new version of Dame

### PR DESCRIPTION
**Pull request by uneeq**

### CODE REVIEW

File: dame_v2.html [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+  <h1>Dame</h1>
+  <span style="color: red;">&hearts;</span>
+  <img src="vector.svg" alt="Dame hearts">
+</article>
+
+<article data-type="card">
+  <h1>Dame</h1>
+  <span style="color: red;">&diams;</span>
+  <img src="vector.svg" alt="Dame diams">
+</article>
```
Comment (oleg-voloshyn) [inline dame_v2.html:23]: @qqumn  Please configure your IDE

File: dame_v2.html [hunk 2]
```diff
@@ -0,0 +1,23 @@
+<article data-type="card">
+  <h1>Dame</h1>
+  <span>&spades;</span>
+  <img src="vector.svg" alt="Dame peaks">
```
Comment (galulex) [inline dame_v2.html:4]: line is too long

File: dame_v2.html [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+
+<article data-type="card">
+  <h1>Dame</h1>
+  <span>&clubs;</span>
+  <img src="vector.svg" alt="Dame clubs">
+</article>
+
+<article data-type="card">
+  <h1>Dame</h1>
+  <span style="color: red;">&hearts;</span>
```
Comment (galulex) [inline dame_v2.html:15]: do not inline css

**Comment by galulex** [comment]
replace the original one

**Pull request closed without merging**